metricsaggregator/src/main/java/io/dockstore/metricsaggregator/MetricsAggregatorAthenaClient.java

```diff
@@ -0,0 +1,342 @@
+                    "projection.entity.values" = "%s",
+                    "projection.registry.type" = "enum",
+                    "projection.registry.values" = "%s",
+                    "projection.org.type" = "injected",
+                    "projection.name.type" = "injected",
+                    "projection.version.type" = "injected",
+                    "projection.platform.type" = "enum",
+                    "projection.platform.values" = "%s",
+                    "storage.location.template" = "s3://%s/${entity}/${registry}/${org}/${name}/${version}/${platform}/"
+                )
+                """, tableName, metricsBucketName, entityProjectionValues, registryProjectionValues, platformProjectionValues, metricsBucketName);
+        executeQuery(query);
+        return tableName;
+    }
+
+    /**
+     * Creates a view for the runExecutions array.
+     * @param entity
+     * @param registry
+     * @param org
+     * @param name
+     * @param version
+     * @return
+     * @throws InterruptedException
+     */
+    public String createRunExecutionsView(String tableName, String entity, String registry, String org, String name, String version) throws InterruptedException {
+        final String viewName = createViewName(entity, registry, org, name, version, "runexecutions");
+        LOG.info("Creating runexecutions view: {}", viewName);
+        final String query = String.format("""
+                CREATE OR REPLACE VIEW "%s" AS
+                WITH dataset AS (
+                    SELECT platform, runexecutions
+                    FROM %s
+                    WHERE entity = '%s' AND registry = '%s' AND org = '%s' AND name = '%s' AND version = '%s'
+                )
+                SELECT platform, unnested.executionid, unnested.dateexecuted, unnested.executionstatus, unnested.executiontime, unnested.memoryrequirementsgb, unnested.cpurequirements, unnested.cost, unnested.region
+                FROM dataset, UNNEST(dataset.runexecutions) AS t(unnested);
+                """, viewName, tableName, entity, registry, org, name, version);
+        executeQuery(query);
+        return viewName;
+    }
+
+    public String createRunExecutionsView(String tableName, S3DirectoryInfo s3DirectoryInfo) throws InterruptedException {
+        return createRunExecutionsView(tableName, s3DirectoryInfo.entityPartition(), s3DirectoryInfo.registryPartition(), s3DirectoryInfo.orgPartition(), s3DirectoryInfo.namePartition(), s3DirectoryInfo.versionPartition());
+    }
+
+    public String createTaskExecutionsView(String entity, String registry, String org, String name, String version) {
+        final String viewName = String.join("/", entity, registry, org, name, version, "taskexecutions");
+        executeQuery(String.format("""
+            CREATE OR REPLACE VIEW '%s' AS
+            WITH dataset AS (
+                SELECT platform, taskexecutions
+                FROM kathys_projected_metrics
```

Comment:
I don't feel strongly about this, but where is this defined?

Reply:
Oops, it was hardcoded while I was testing, so I'll update it to parameters, but note that this isn't actually called anywhere